pageStart.js

```diff
@@ -0,0 +1,26 @@
+import { Selector, t } from 'testcafe';
+
+export default class StartPage {
+constructor(){
+this.themeTitle = /anime/g;
```

Comment:
`themeTitle` and `storeTitle` should be stored in the test itself as constants. they are test data and should be passed into the function from the test

although in my opinion it would be more correctly to select a random theme from the list of available themes on the page